======================= Schemathesis test session starts =======================
Schema location: file:///mnt/d/ag/github/lazarus_target_apis/data/api_petstore/api_petstore.json
Base URL: https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0
Specification version: Open API 3.0.0
Random seed: 19475166661818775736462516568308055448
Workers: 1
Collected API operations: 20
Collected API links: 0
API probing: ...API probing: SUCCESS
Schema analysis: ...Schema analysis: SKIP

POST /AAYUSHGARGBU/api_petstore/1.0.0/pet E                               [  5%]
PUT /AAYUSHGARGBU/api_petstore/1.0.0/pet F                                [ 10%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus F                   [ 15%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags F                     [ 20%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                        [ 25%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                       [ 30%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                     [ 35%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId}/uploadImage F           [ 40%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/store/inventory F                    [ 45%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/store/order F                       [ 50%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} F              [ 55%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} F           [ 60%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user .                              [ 65%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user/createWithArray .              [ 70%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user/createWithList .               [ 75%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/login F                         [ 80%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/logout .                        [ 85%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                    [ 90%]
PUT /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                    [ 95%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                 [100%]

==================================== ERRORS ====================================
__________________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
Runtime Error

'latin-1' codec can't encode character '\u0100' in position 0: Body ('Ā') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\u0100' in position 0: Body ('Ā') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Runtime Error

'latin-1' codec can't encode characters in position 1-2: Body ('\U00040fbe\U00053fc) is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode characters in position 1-2: Body ('\U00040fbe\U00053fc) is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Runtime Error

'latin-1' codec can't encode character '\U0004e103' in position 0: Body ('\U0004e103') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\U0004e103' in position 0: Body ('\U0004e103') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Runtime Error

'latin-1' codec can't encode character '\U00053fca' in position 6: Body ('\U00053fca') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\U00053fca' in position 6: Body ('\U00053fca') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Need more help?
    Join our Discord server: https://discord.gg/R9ASRAmHnA
=================================== FAILURES ===================================
__________________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
1. Test Case ID: 71kQM2

- Undocumented HTTP status code

    Received: 429
    Documented: 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/xml' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet

___________________ PUT /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
1. Test Case ID: AG7wtc

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404, 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X PUT -H 'Content-Type: application/json' -d '{"name": "doggie", "photoUrls": []}' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet

____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus _____________
[FLAKY] Schemathesis was not able to reliably reproduce this failure

1. Test Case ID: eK5NVp

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET 'https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus?status=sold'

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags ______________
[FLAKY] Schemathesis was not able to reliably reproduce this failure

1. Test Case ID: WM06ch

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags

_______________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} _______________
1. Test Case ID: Hd00UJ

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET -H 'api_key: [Filtered]' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

______________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} _______________
1. Test Case ID: UbGE6M

- Undocumented HTTP status code

    Received: 429
    Documented: 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/x-www-form-urlencoded' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

_____________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} ______________
1. Test Case ID: AbgtO3

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId}/uploadImage _________
1. Test Case ID: cbgf6b

- Undocumented HTTP status code

    Received: 429
    Documented: 200

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0/uploadImage

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/store/inventory _____________
1. Test Case ID: jvF0m1

- Undocumented HTTP status code

    Received: 429
    Documented: 200

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET -H 'api_key: [Filtered]' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/inventory

______________ POST /AAYUSHGARGBU/api_petstore/1.0.0/store/order _______________
1. Test Case ID: KhgVmC

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/json' -d '{}' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order

__________ GET /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} __________
1. Test Case ID: sLGR1J

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order/1

________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} _________
1. Test Case ID: 7Y3w7o

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order/1

_______________ GET /AAYUSHGARGBU/api_petstore/1.0.0/user/login ________________
1. Test Case ID: 95dGwA

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET 'https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/login?password=&username='

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} _____________
1. Test Case ID: f1m4On

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

_____________ PUT /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} _____________
1. Test Case ID: SkMfYo

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X PUT -H 'Content-Type: application/json' -d '{}' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

___________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} ____________
1. Test Case ID: 38KLXf

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

=================================== SUMMARY ====================================

Performed checks:
    not_a_server_error                              455 / 455 passed          PASSED 
    status_code_conformance                         315 / 455 passed          FAILED 
    content_type_conformance                        455 / 455 passed          PASSED 
    response_headers_conformance                    455 / 455 passed          PASSED 
    response_schema_conformance                     455 / 455 passed          PASSED 

Note: Use the 'X-Schemathesis-TestCaseId' header to correlate test case ids from failure messages with server logs for debugging.

Note: To replicate these test failures, rerun with `--hypothesis-seed=19475166661818775736462516568308055448`

Tip: Use the `--report` CLI option to visualize test results via Schemathesis.io.
We run additional conformance checks on reports from public repos.

=================== 4 passed, 16 failed, 1 errored in 59.73s ===================
